build.sbt

```diff
@@ -384,9 +390,25 @@ lazy val engineStandalone = (project in engine("standalone/engine")).
   ).
   dependsOn(interpreter % "provided", standaloneUtil, httpUtils % "provided", testUtil % "it,test")
 
+lazy val standaloneDockerSettings = {
+  val workingDir = "/opt/nussknacker"
+
+  commonDockerSettings ++ Seq(
+    dockerEntrypoint := Seq(s"$workingDir/bin/nussknacker-standalone-entrypoint.sh", dockerPort.toString),
```

Comment:
I think that this port argument isn't handled. I see that by default exposed ports are:
```
      - "8070:8070"
      - "8071:8080"
```
IMO we don't need to "expose" it by entrypoint but can use standard Dockerfile expose directive. It will help developers find out on which ports are available and some comment will help to find out which one is for what.

Reply:
I'll add both processes and management ports through `dockerExposedPorts`